### PR TITLE
Added feature to position snackbar on the screen (Android)

### PR DESCRIPTION
**Pull request by imMayankRaj**
Plus working on #70

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -44,6 +51,7 @@ const SnackBar: ISnackBar = {
       options.backgroundColor = processColor(options.backgroundColor);
     }
 
+    console.log(options);
```
Comment (cooperka) [inline src/index.js]: Please remove log

File: src/index.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ type Action = {
 type SnackBarOptions = {
   title: string,
   duration?: number,
+  gravity?:number,
```
Comment (cooperka) [inline src/index.js]: Space after colon

File: src/index.d.ts [hunk 3]
```diff
@@ -39,6 +39,12 @@ interface SnackBarOptions {
    */
   duration?: number;
 
+  /**
+   * Defines the postion of snackbar on the screen.
+   * Must be one of Snackbar.GRAVITY_TOP, Snackbar.GRAVITY_BOTTOM, or Snackbar.GRAVITY_CENTER.
+   */
+
+  gravity?:number;
```
Comment (cooperka) [inline src/index.d.ts]: - Put comment directly above property (no extra newline)
- Add newline below property
- Add space after colon

File: android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java [hunk 4]
```diff
@@ -115,6 +120,13 @@ public void onClick(View v) {
             snackbar.setActionTextColor(actionDetails.getInt("color"));
         }
 
+        if(options.hasKey("gravity")) {
```
Comment (cooperka) [inline android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java]: Space after `if`

File: android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java [hunk 5]
```diff
@@ -115,6 +120,13 @@ public void onClick(View v) {
...
+            View snackbarView = snackbar.getView();
+            FrameLayout.LayoutParams params =(FrameLayout.LayoutParams)snackbarView.getLayoutParams();
```
Comment (cooperka) [inline android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java]: Space after `=`

**Review by cooperka — CHANGES_REQUESTED** [review]
Just a few nits for consistency 😸 looks good overall!

**Comment by cooperka** [comment]
Hi @imMayankRaj, thanks for this. Does the animation look good when it's at the top? In other words, does the snackbar slide down instead of up when you position it at the top? If so I'd be happy merging on 2 conditions:

1. Please update README to document new props
2. File a ticket to add this prop to iOS as well. You don't have to do it yourself, this is more of a note to myself.

**Comment by imMayankRaj** [comment]
Hey @cooperka, changing the gravity of the Snackbar has no effect on the animation. It's still the same.

**Comment by cooperka** [comment]
Hmm, doesn't that look quite strange? I think we'll need to find a different solution in that case. Thoughts?

**Comment by imMayankRaj** [comment]
Sorry, my bad, it's working as expected i.e it's sliding down when at the bottom and sliding up when at the top. And also the view gets dismissed as a whole i.e without any sliding animation, when at the center.

**Comment by cooperka** [comment]
I tried to push updated code to your branch and GitHub seems to have reacted by closing this PR automatically. I'm unable to re-open. Can you please do this on your machine:

```console
cd react-native-snackbar/
git push origin 9428680:master -f
```

That will restore your latest commit back to your branch. Apologies for the inconvenience; hopefully that allows me to re-open the PR.

**Comment by cooperka** [comment]
I also tested it on my emulator and the animation isn't behaving as expected. Can you post a screen recording and some info about your device version? I'm expecting the Snackbar to slide in and out from the top as you described.

Here's what I see, no change in animation when positioned at the top:

animation of snackbar with gravity

**Comment by imMayankRaj** [comment]
ezgif com-video-to-gif

It's working fine on my device.
I am using a physical device.
MOTO g5 plus
Android 8.1

**Comment by imMayankRaj** [comment]
> 
> 
> I tried to push updated code to your branch and GitHub seems to have reacted by closing this PR automatically. I'm unable to re-open. Can you please do this on your machine:
> 
> ```
> cd react-native-snackbar/
> git push origin 9428680:master -f
> ```
> 
> That will restore your latest commit back to your branch. Apologies for the inconvenience; hopefully that allows me to re-open the PR.

Done. Try again

**Comment by cooperka** [comment]
> Try again

No luck :crying_cat_face: I've never encountered this before. No worries, I can merge manually later on.

> It's working fine on my device

The animations don't seem to be coming through from your screen recorder. I just see it flash onto the screen with no animation.

**Comment by imMayankRaj** [comment]
No, it's working as taken by recorder. That was the reason I was confused. The animation doesn't come up, on my device, as you are expecting.

**Comment by cooperka** [comment]
Strange! I wonder if your animations are disabled somehow. Unfortunately I won't be able to merge this PR until the animation happens properly... I'm not sure if Snackbar would support that easily or not.

**Pull request closed without merging**